<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>Поиск...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="recipeListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
      <item>
       <widget class="ViewImages" name="widgetImgs" native="true"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="descriptTextEdit">
        <property name="placeholderText">
         <string>Описание рецепта...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actSaveBook"/>
    <addaction name="actSaveBookAs"/>
    <addaction name="actLoadBook"/>
    <addaction name="separator"/>
    <addaction name="actExportRec"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <widget class="QMenu" name="menuSearch">
     <property name="title">
      <string>Поиск</string>
     </property>
     <addaction name="actSearchTag"/>
     <addaction name="actSearchIngrs"/>
     <addaction name="actSearchExt"/>
    </widget>
    <addaction name="actAddRec"/>
    <addaction name="separator"/>
    <addaction name="menuSearch"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actSaveBook">
   <property name="text">
    <string>Сохранить книгу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actLoadBook">
   <property name="text">
    <string>Загрузить книгу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actExportRec">
   <property name="text">
    <string>Экспортировать рецепт</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actAddRec">
   <property name="text">
    <string>Добавить рецепт</string>
   </property>
  </action>
  <action name="actSearchTag">
   <property name="text">
    <string>Поиск по тэгу</string>
   </property>
  </action>
  <action name="actSearchIngrs">
   <property name="text">
    <string>Поиск по ингридиентам</string>
   </property>
  </action>
  <action name="actSearchExt">
   <property name="text">
    <string>Расширенный поиск</string>
   </property>
  </action>
  <action name="actSaveBookAs">
   <property name="text">
    <string>Сохранить книгу как...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewImages</class>
   <extends>QWidget</extends>
   <header location="global">viewimages.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
